repository/src/main/java/org/alfresco/repo/admin/patch/PatchServiceImpl.java

```diff
@@ -322,15 +322,15 @@ private boolean applyPatchAndDependencies(final Patch patch, Map<String, Applied
     }
     
     private AppliedPatch applyPatch(Patch patch)
-    {
+    {
         PatchWork work = new PatchWork(patch);
         work.setAppliedPatchDAO(appliedPatchDAO);
         work.setTransactionService(transactionService);
         work.setAppliedPatchDAO(appliedPatchDAO);
         work.setDescriptorService(descriptorService);
         work.setLogger(logger);
         work.execute();
-
+
         return work.getAppliedPatch();
     }
 

```

Comment:
update license header from 2016

Reply:
I just removed this change, it was irrelevant, so I won't touch it :)